graf3d/eve/CMakeLists.txt

```diff
@@ -228,7 +228,9 @@ if(MSVC)
       OpenGL::GL
       OpenGL::GLU
       ${FTGL_LIBRARIES}
-      ${GLEW_LIBRARIES}
+      GLEW
```

Comment:
Why do we need to repeat `GLEW`? This looks like a deficiency in `ROOT_STANDARD_PACKAGE` checking whether a dependency is a builtin?